grapher/captionedChart/CaptionedChart.tsx

```diff
@@ -79,7 +79,7 @@ interface CaptionedChartProps {
 }
 
 const OUTSIDE_PADDING = 15
-const PADDING_BELOW_HEADER = 18
+const PADDING_BELOW_HEADER = 16
```

Comment:
Decreased the padding to compensate a bit for the subtitle `height` being inaccurate – it used to not take into account `lineHeight` correctly, it was an underestimate.